luminoso_api/json_stream.py

```diff
@@ -254,6 +260,13 @@ def _read_csv(reader, header, encode_fn):
         yield row_dict
 
 
+def convert_date(date_string, date_format):
+    if date_format != 'epoch':
+        return datetime.datetime.strptime(date_string, date_format).timestamp()
```

Comment:
OK, sure.  It absolutely shouldn't do so quietly; you don't want to tell it "my dates are ISO" when they're actually not, and then have the script go "hm, can't parse any of these, I'll just pretend there are no dates at all" without telling the user.  Conversely, though, I'd bet--Tim can (dis)confirm?--that it's not unheard of for a file to have one date that's not quite right.  Bear in mind I'm not even anticipating "Barney the Purple Dinosaur", but rather "12:23" instead of "12:23:34" or "1-23-2015" instead of "1/23/2015".